mix.exs

```diff
@@ -13,7 +13,7 @@ defmodule BitstylesPhoenix.MixProject do
       package: package(),
       source_url: "https://github.com/bitcrowd/bitstyles_phoenix",
       docs: [
-        main: "BitstylesPhoenix",
+        main: "readme",
```

Comment:
?

Reply:
It changes the front of the hex_docs to be the README.md instad of the main module. 
Weirdly enough it issues a warning when this is `README` (even tough it works) and does not work when this is `README.html`